docs/guides/overview/why-cypress.mdx

```diff
@@ -225,11 +232,87 @@ it('adds a todo', () => {
 ### Other
 
 Finally, through a large number of [official and 3rd party plugins](/plugins)
-you can write Cypress [a11y](https://github.com/component-driven/cypress-axe),
+you can write other types of tests.
+
+## Get Started
+
+### <Icon name="cog" /> Set up tests
+
+There are no servers, drivers, or any other dependencies to install or
+configure. You can quickly see your first passing test within minutes for
+[End-to-end tests](/guides/end-to-end-testing/writing-your-first-end-to-end-test) or
+[Component tests](/guides/component-testing/getting-started).
 
-<a href="/plugins#visual-testing">visual</a>,
-[email](/faq/questions/using-cypress-faq#How-do-I-check-that-an-email-was-sent-out)
-and other types of tests.
+### <Icon name="code" /> Write tests
+
+Tests written in Cypress are meant to be easy to read and understand. Our [API](/api/table-of-contents)
+comes fully baked, on top of tools you are familiar with already.
+
+<DocsVideo src="/img/snippets/writing-tests.mp4" title="Writing Tests" />
+
+### <Icon name="play-circle" /> Run tests
+
+Cypress runs as fast as your browser can render content. You can watch tests run
+in real time as you develop your applications and time travel to previous steps. TDD FTW!
```

Comment:
"TDD FTW!"

You may want to spell this out (or leave it out), as the TLA "FTW" has multiple interpretations.

Reply:
Yah, I thought this wording was a bit....out of place also...